friend_service/friend_service_grpc.py

```diff
@@ -1,13 +1,18 @@
-from concurrent import futures
+import instana
```

Comment:
all looks good, why do we need to import instana?

Reply:
This is to manually activate `instana` instrumentation. It won't take effect if instana agent is not installed. The Instana auto-injection doesn't seem to work for me, so I follow their instruction to enable manual injection.